### DISCUSSION TITLE
Is piped figure attribute supported in USFM3?

### DISCUSSION THREAD

**Discussion by kavitharaju** [usfm-bible/tcdocs#32] [General]
This test case contains the older usage of figure marker with multiple pipes. Is this  still to be supported?
https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/paratextTests/EmptyFigure/origin.usfm#L7

**Comment by KentSpiel** [comment]
No, this is a major change. The Pipe syntax is deprecated in 3.2 and will be invalid in 4.0. https://docs.usfm.bible/usfm/latest/fig/fig.html
